### PR TITLE
added helm2 page

### PR DESCRIPTION
**Pull request by jesse-codefresh**

### CODE REVIEW

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm2/docs/

File: _docs/new-helm/helm2-support.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+---
+
+**Sunsetting Helm 2 Support**
+
+At Codefresh, we support both Helm 2 and Helm 3 releases, both in the graphical dashboards and in pipeline steps.
+
+However, the Helm team has discontinued all Helm support since November 2020. 
+This means; no new Helm 2 releases (not even for security updates). “Stable” and “Incubator" repositories will be discontinued.
+
+https://github.com/helm/charts
```
Comment (kostis-codefresh) [inline _docs/new-helm/helm2-support.md:15]: This URL should be part of 3 bullet points (no new helm 2 releases, stable and incubator discontinued)

File: _docs/new-helm/helm2-support.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+For migrating your Helm 2 charts you can use the [helm-2to3 plugin](https://github.com/helm/helm-2to3)
+
+Codefresh will also discontinue support for Helm 2 on July 16 2021. After this date, all Codefresh Helm dashboards including the Helm releases, and promotions will stop supporting Helm 2 clusters and namespaces. The Codefresh Helm [step](https://codefresh.io/steps/step/helm) will no longer support Helm 2.
+
+Please contact us at support@codefresh.io if you have any questions or concerns.
+
+What to read next:
```
Comment (kostis-codefresh) [inline _docs/new-helm/helm2-support.md:23]: see how the other pages in the documentation have a section "what to read next" and make it look the same with them.

File: _docs/new-helm/helm2-support.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+At Codefresh, we support both Helm 2 and Helm 3 releases, both in the graphical dashboards and in pipeline steps.
+
+However, the Helm team has discontinued all Helm support since November 2020. 
+This means; no new Helm 2 releases (not even for security updates). “Stable” and “Incubator" repositories will be discontinued.
+
+https://github.com/helm/charts
+
+For migrating your Helm 2 charts you can use the [helm-2to3 plugin](https://github.com/helm/helm-2to3)
+
+Codefresh will also discontinue support for Helm 2 on July 16 2021. After this date, all Codefresh Helm dashboards including the Helm releases, and promotions will stop supporting Helm 2 clusters and namespaces. The Codefresh Helm [step](https://codefresh.io/steps/step/helm) will no longer support Helm 2.
```
Comment (kostis-codefresh) [inline _docs/new-helm/helm2-support.md:19]: Add documentation links for Helm releases and Helm promotions.

File: _docs/new-helm/helm2-support.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Sunsetting Helm 2 Support"
+description: "Migrate Your Applications from Helm 2 to Helm 3"
+group: new-helm
+toc: true
+---
+
+**Sunsetting Helm 2 Support**
```
Comment (kostis-codefresh) [inline _docs/new-helm/helm2-support.md:8]: This is already the title of the page. Remove this.

File: _docs/new-helm/helm2-support.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+At Codefresh, we support both Helm 2 and Helm 3 releases, both in the graphical dashboards and in pipeline steps.
+
+However, the Helm team has discontinued all Helm support since November 2020.
```
Comment (kostis-codefresh) [inline _docs/new-helm/helm2-support.md:12]: November 2020 should be a link to https://helm.sh/blog/helm-v2-deprecation-timeline/

File: _docs/new-helm/helm2-support.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+For migrating your Helm 2 charts you can use the [helm-2to3 plugin](https://github.com/helm/helm-2to3)
+
+Codefresh will also discontinue support for Helm 2 on July 16 2021. After this date, all Codefresh Helm dashboards including the Helm releases, and promotions will stop supporting Helm 2 clusters and namespaces. The Codefresh Helm [step](https://codefresh.io/steps/step/helm) will no longer support Helm 2.
+
+Please contact us at support@codefresh.io if you have any questions or concerns.
+
+What to read next:
+
+[Helm Quick Start Guide](https://codefresh.io/docs/docs/getting-started/helm-quick-start-guide/)
```
Comment (kostis-codefresh) [inline _docs/new-helm/helm2-support.md:25]: Convert all these links to use the special syntax with site.baseurl instead of being absolute links

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm2/docs/

File: _docs/new-helm/helm2-support.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+Codefresh will also discontinue support for Helm 2 on July 16 2021. After this date: 
+1. All Codefresh Helm dashboards including the [Helm releases](https://codefresh.io/docs/docs/new-helm/helm-releases-management/), and [promotions](https://codefresh.io/docs/docs/new-helm/helm-environment-promotion/) will stop supporting Helm 2 clusters and namespaces. 
+2. The Codefresh Helm [step](https://codefresh.io/steps/step/helm) will no longer support Helm 2.
+
+Please contact us at support@codefresh.io if you have any questions or concerns.
+
+## What to read next
+
+* [Helm Quick Start Guide](https://codefresh.io/docs/docs/getting-started/helm-quick-start-guide/)
+
```
Comment (kostis-codefresh) [inline _docs/new-helm/helm2-support.md:29]: Remove empty lines

File: _docs/new-helm/helm2-support.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+At Codefresh, we support both Helm 2 and Helm 3 releases, both in the graphical dashboards and in pipeline steps.
+
+However, the Helm team has discontinued all Helm support since [November 2020](https://helm.sh/blog/helm-v2-deprecation-timeline/). 
+Therefore:
+* No new Helm 2 releases (not even for security updates)
+* “Stable” and “Incubator" repositories will be discontinued
+* https://github.com/helm/charts is now obsolete
+
+https://github.com/helm/charts
```
Comment (kostis-codefresh) [inline _docs/new-helm/helm2-support.md:16]: Remove this line

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm2/docs/

**Comment by itai-codefresh** [comment]
[CI] Staging environment is at http://staging-docs.codefresh.io.s3-website.us-east-1.amazonaws.com/helm2/docs/

**Pull request merged**